src/EA.Iws.DataAccess/Repositories/Imports/ImportNotificationTransactionRepository.cs

```diff
@@ -32,5 +32,17 @@ public void Add(ImportNotificationTransaction transaction)
         {
             context.ImportNotificationTransactions.Add(transaction);
         }
+
+        public async Task DeleteById(Guid transactionId)
+        {
+            var transaction = await context.ImportNotificationTransactions.Where(t => t.Id == transactionId).SingleAsync();
+
+            context.ImportNotificationTransactions.Remove(transaction);
```

Comment:
You can use `context.DeleteOnCommit(transaction);` instead of `context.ImportNotificationTransactions.Remove(transaction);`. I think the performance is slightly better.